omero_search_engine/__init__.py

```diff
@@ -22,12 +22,19 @@
 import logging
 from elasticsearch import Elasticsearch
 from flasgger import Swagger, LazyString, LazyJSONEncoder
-from omero_search_engine.database.database_connector import DatabaseConnector
+from watchdog.observers import Observer
+from watchdog.events import FileSystemEventHandler
+import threading
+import time
+
+# from omero_search_engine.database.database_connector import DatabaseConnector
 from omero_search_engine.__version__ import __version__
+
 from configurations.configuration import (
     configLooader,
     load_configuration_variables_from_file,
     set_database_connection_variables,
+    get_configure_file,

```

Comment:
Do you ``get_configuration_file``?

Reply:
It retrieves the file name to monitor changes and reloads it when necessary

Reply:
So the name of the method should be ``get_configuration_file``?